src/catalog/catalog_manager.py

```diff
@@ -256,6 +256,20 @@ def create_udf(self, name: str, impl_file_path: str,
         self._udf_io_service.add_udf_io(udf_io_list)
         return metadata
 
+    def delete_metadata(self, table_name: str) -> bool:
```

Comment:
Is this the only function required to drop a created udf? And table_name = udf_name?

Reply:
No, this is to delete the table metadata. I will also add one for the UDF.